warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__no_rt_critical_validation_errors.sql

```diff
@@ -39,8 +39,8 @@ int_gtfs_quality__no_rt_critical_validation_errors AS (
         rt_files,
         errors,
         CASE
-            WHEN COALESCE(errors, 0) = 0 THEN "PASS"
-            WHEN rt_files IS NOT NULL THEN "FAIL" -- we had no errors, but did have files
+            WHEN rt_files IS NOT NULL AND COALESCE(errors, 0) = 0 THEN "PASS"
+            WHEN rt_files IS NOT NULL AND errors > 0 THEN "FAIL" -- we had no errors, but did have files
```

Comment:
Nit pick that I think the comment here is still inaccurate, but actual logic now correct

Reply:
thank you!